org.eclipse.m2e.tests.common/META-INF/MANIFEST.MF

```diff
@@ -21,7 +21,13 @@ Bundle-RequiredExecutionEnvironment: JavaSE-17
 Bundle-Vendor: Eclipse.org - m2e
 MavenArtifact-GroupId: org.eclipse.m2e
 MavenArtifact-ArtifactId: org.eclipse.m2e.tests.common
-Export-Package: org.eclipse.m2e.tests.common;x-friends:="org.eclipse.m2e.tests,org.eclipse.m2e.core.tests,org.eclipse.m2e.jdt.tests,org.eclipse.m2e.core.ui.tests,org.eclipse.m2e.wtp.tests"
+Export-Package: org.eclipse.m2e.tests.common;
+  x-friends:="org.eclipse.m2e.tests,
+   org.eclipse.m2e.core.tests,
```

Comment:
Since this plug-in itself is only for tests, I think it would be ok to define other test-plugins as friends. But if you disable discouraged access warnings there that is probably not necessary.

Reply:
Okay, will disable it in all test bundles, and then apply only the remaining fixes in non test bundles.